textinput_model.go

```diff
@@ -30,114 +30,131 @@ func (p *textinputPlugin) addChar(char []rune) {
 
 	p.selectionBase += len(char)
 	p.selectionExtent = p.selectionBase
-	p.updateEditingState()
 }
 
-func (p *textinputPlugin) MoveCursorHome(modsIsModifier bool, modsIsShift bool, modsIsWordModifierShift bool, modsIsWordModifier bool) {
+func (p *textinputPlugin) MoveCursorHomeSimple() {
```

Comment:
those methods don't needs to be exported.
`s/MoveCursorHomeSimple/moveCursorHomeSimple`